aliasing/api/character.py

```diff
@@ -163,6 +165,49 @@ def create_cc(self, name: str, *args, **kwargs):
             self.delete_cc(name)
         return self.create_cc_nx(name, *args, **kwargs)
 
+    def edit_cc(self, name: str, minVal: str = None, maxVal: str = None, reset: str = None,
+                dispType: str = None, reset_to: str = None, reset_by: str = None,
+                title: str = None, desc: str = None):
+        """
+        Edits an existing custom counter.
+
+        Pass ``None`` to remove an argument entirely.
+        Will clamp counter value to new limits if needed.
+
+        :param str name: The name of the counter to edit.
+        :param str minVal: The minimum value of the counter. Supports :ref:`cvar-table` parsing.
+        :param str maxVal: The maximum value of the counter. Supports :ref:`cvar-table` parsing.
+        :param str reset: One of ``'short'``, ``'long'``, ``'hp'``, ``'none'``, or ``None``.
+        :param str dispType: Either ``None`` or ``'bubble'``.
+        :param str reset_to: The value the counter should reset to. Supports :ref:`cvar-table` parsing.
+        :param str reset_by: How much the counter should change by on a reset. Supports dice but not cvars.
+        :param str title: The title of the counter.
+        :param str desc: The description of the counter.
+        :rtype: AliasCustomCounter
+        :raises: :exc:`ConsumableException` if the counter does not exist.
+        :returns: The edited counter
+        """
+        counter = self._get_consumable(name)
+
+        minVal = optional_cast_arg_or_default(minVal, default=counter.min)
+        maxVal = optional_cast_arg_or_default(maxVal, default=counter.max)
+        reset = optional_cast_arg_or_default(reset, default=counter.reset)
+        dispType = optional_cast_arg_or_default(dispType, default=counter.display_type)
+        reset_to = optional_cast_arg_or_default(reset_to, default=counter.reset_to)
+        reset_by = optional_cast_arg_or_default(reset_by, default=counter.reset_by)
+        title = optional_cast_arg_or_default(title, default=counter.title)
+        desc = optional_cast_arg_or_default(desc, default=counter.desc)
+
+        if self.cc_exists(name):
```

Comment:
If we are raising `ConsumableException` before we get here, this is redundant (and we can remove a level of indentation!)